.circleci/config.yml

```diff
@@ -22,27 +22,28 @@ jobs:
       # build.
       - image: "nixorg/nix:circleci"
 
-    resource_class: "xlarge"
```

Comment:
Let's leave this.  If the cache ever misses then w/o this I think we have a guaranteed build failure.  Also, having builds complete faster even in the case of a cache hit (because our software builds faster) seems fine to me.